py/torch_tensorrt/dynamo/conversion/impl/__init__.py

```diff
@@ -12,6 +12,7 @@
     elementwise,
     embedding,
     grid,
+    index,
```

Comment:
This can likely be removed - it seems to be causing a circular import error in CI

Reply:
Thanks! It seems I overlooked removing an unnecessary import.